package.json

```diff
@@ -74,9 +74,6 @@
         "start-react": "export BROWSER=none REACT_EDITOR=none && react-app-rewired start",
         "test": "react-app-rewired test"
     },
-    "eslintConfig": {
```

Comment:
why do we want to drop this?

Reply:
afaict it has no effect and is entirely overridden by the eslintrc.json file. I wasted an hour trying to fix this using this json block before I realized what was going on. I figured removing it may spare someone down the road from doing the same.